src/main/java/com/feywild/feywild/events/loot_modifiers/MineshaftChestLootModifier.java

```diff
@@ -57,18 +54,15 @@ protected MineshaftChestLootModifier(LootItemCondition[] conditionsIn, Item addi
         @Override
         public MineshaftChestLootModifier read(ResourceLocation location, JsonObject object, LootItemCondition[] ailootcondition) {
 
-            Item addition = ForgeRegistries.ITEMS.getValue(
-                    new ResourceLocation(GsonHelper.getAsString(object, "addition")));
-
-            return new MineshaftChestLootModifier(ailootcondition, addition);
+            return new MineshaftChestLootModifier(ailootcondition);
 
         }
 
         @Override
         public JsonObject write(MineshaftChestLootModifier instance) {
 
             JsonObject json = makeConditions(instance.conditions);
-            json.addProperty("addition", ForgeRegistries.ITEMS.getKey(instance.addition).toString());
+            json.addProperty("addition", Objects.requireNonNull(ForgeRegistries.ITEMS.getKey(ItemStack.EMPTY.getItem())).toString());
             return json;
```

Comment:
Serialising an empty item is not required.

```suggestion
            return makeConditions(instance.conditions);
```